sunpy/net/dataretriever/sources/goes.py

```diff
@@ -22,7 +22,22 @@ class XRSClient(GenericClient):
     """
     Provides access to the GOES XRS fits files archive.
 
-    Searches data hosted by the `Solar Data Analysis Center <https://umbra.nascom.nasa.gov/goes/fits/>`__.
+    Searches data for the GOES XRS data both on NASA servers prior
```

Comment:
both? I thought we defaulted to just new servers?

Reply:
yep - but for the data prior to GOES 13 the NASA server data is correct (NOAA haven't provided re-processed data)